### DISCUSSION TITLE
Tailwind classes are not working after installation of tanstack router

### DISCUSSION THREAD

**Discussion by solomonrazak** [tailwindlabs/tailwindcss#14430] [Help]
my tailwind classes are not working again after I installed tanstack router. was working initially but installing tanstack router it isnonly the second time I am using tanstackt. this is  in my project and both times I faced same isssues

**Comment by Xstyler85** [comment]
I had the same problem, then I found this in one of the tanstack router examples:

**postcss.config.cjs**
```
module.exports = {
    plugins: [
      require('tailwindcss/nesting'),
      require('tailwindcss'),
      require('autoprefixer'),
    ],
}
 ```
